.github/workflows/bazel_test.yml

```diff
@@ -1,14 +1,32 @@
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
 name: Bazel Test
 
 on: [push, pull_request]
 
 jobs:
-  linux:
+  bazel_test:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
       - name: Run all tests
         working-directory: ./
+        # Run all bazel test but only need to build //examples/...
         run: |
-          bazel test //...
+          bazel test --config=clang //... -- -//examples/...
```

Comment:
One way to get around "exclude //examples/..." is to instead exclude all `cc_fuzz_test` by specifying them with a tag and using `--test_tag_filters` (https://docs.bazel.build/versions/master/command-line-reference.html#flag--test_tag_filters). 

You can also specify just build fuzz tests with the build tag filter https://docs.bazel.build/versions/master/command-line-reference.html#flag--build_tag_filters

Reply:
TIL about `--test_tag_filters` :)